site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -10,10 +10,10 @@
     <hr>
 
     <div class="dropdown-wrapper">
-        <div onclick='updateDropdownStates($(this).find(".dropdown-arrow").hasClass("down"), "today_polls_dropdown", "{{ base_url }}")' class="dropdown-bar">
+        <div class="dropdown-bar">
             <h2> Today's Polls </h2>
             <div class="dropdown-btn">
-                <i class="dropdown-arrow{{dropdown_states['today'] ? ' down' : ' right'}}"></i>
```

Comment:
This looks nice in dark mode but removes a lot of functionality. In master, a cookie is created for future, today, and old which tell the website if the panels should be open or not as well as the chevrons on page load. I think this feature is still useful so the panels don't have to be hidden every time the page is loaded. I honestly think it can be kept the way it was before but only the color should be changed through CSS. Otherwise some refactoring may need to be down.

Reply:
Ok, I'll update.  I didn't notice any differences so I just removed the code initially.

Reply:
Yeah in order to see the difference you need to open/close the panels then refresh. If you compare the functionality of that to master you will see the difference.

Reply:
PR updated.